### ISSUE TITLE
REX Huge Story

### ISSUE THREAD

**Issue by PxxBxx** [marian-m12l/studio#293]
Hello, it's not an issue, but a REX of the last 3 hours I spent doing a massive story for my kid's birthday.

All in all, I did setup :

- one action with 49 choices on the first level
- 49 stories based on 1-5 podcast each
- backward (Home) links on each intermediate scene
- chain link between all stories (at the end of a path you autoplay the next path)

The editor managed :

- 100+ actions
- 100+ scenes

The final package is 500MB !
And of course, it's perfectly playing on the Lunii device

Why I bother telling you ? BECAUSE IT WORKED WITHOUT ANY CRASH ! :)

**Thanks a lot**

PS : to make all the small audio files for the introduction, I found an easy way :
- you list (write down) all the titles
- you find a quiet place
- you start any voice-recording app on your phone
- you read aloud your list, with a 1sec pause between each titles
- you export the mp3/mp4
- ` ffmpeg -i YOUR.MP3 -af silencedetect=d=0.5 -f null -` <= list all the start/end of the silences longer than 0.5sec
- ` ffmpeg -i YOUR.MP3 -ss START_VALUE --to END_VALUE EXTRACT.MP3`  <= extract a sequence, you loop with all the above values, and you get all your small MP3 files !

PPS : a screenshot of the editor O.o

<img width="394" alt="image" src="https://user-images.githubusercontent.com/6170432/174901015-8cf0413b-ca0a-473a-ae26-df826dfe93f1.png">

**Comment by LHommeQuAVuLOurs** [comment]
Hello, Thank you for your feedback,
I created a "big pack" like that once and experienced some random problems with my lunii : sometimes it just stopped reading any menu sound, and only read the fist seconds of the stories (any story of any pack was impacted, even the standard one included). A reboot of the lunii didn't make it better, nor did a "reset" from the lunii app. The only thing which solved this bug was to format the lunii as if it was a SD card, and then reset it from the lunii desktop app. Importing the big pack again triggers the bug but not necessarily the first day, it can work 2/3 days and then it's over.

I detected that smaller packs without chain links didn't trigger the bug so I kept going with small packs.

I'd love to borrow you your pack to see what options you used and compare it to mine see if there's any difference, and also if you can send me the pacakage si I try to load it on the lunii and check if it triggers my bug or not ?

Thank you very much !

**Comment by octplane** [comment]
@PxxBxx hey, nice feedback! I'm planning on doing something like that, but simpler (maybe) using downloaded Podcasts. Have you assembled everything by hand or have you scripted the whole Pack generation? I'm asking because in my case, I'll be starting with a RSS feed for the podcast and would like to automate stuff as much as possible... Thanks! And awesome screenshot too!

**Comment by octplane** [comment]
Replying to myself, another issue mentions several answers to my question: https://github.com/marian-m12l/studio/issues/246#issuecomment-933803207

**Comment by PxxBxx** [comment]
@octplane Thank you, and THANK YOU !
I did not see the project you linked on the #246  comment, but I just look at it and say WHAW! amazing job, it seems really nice.
I had my own scripts (rss -> fetch -> generate titles with text to speech -> forge .json), but it was a mess (I'm not a modern developper).
I think next time I'll try the script from jersou (I already starred it, it totally deserves it at least for the Readme that is delightful)

**Comment by PxxBxx** [comment]
But I just want to remind the feedback from @LHommeQuAVuLOurs : HUGE packs may be risky.
In my scripts I did some pagination, the RSS (and/or folders) were split (eg. 20 stories on the first node, they may have many parts, but I limited my entries to 20 on the navigation, after seeing my daughter turning the knob like a devil, if it may trigger a software bug, don't forget that the knob can also broke...).
For example I have "OLI partie 1", "OLI partie 2" and so on depending on the RSS length.
